pom.xml

```diff
@@ -139,7 +139,6 @@
             <groupId>javax.transaction</groupId>
             <artifactId>jta</artifactId>
             <version>${jta.api.version}</version>
-            <scope>provided</scope>
```

Comment:
Does this still need to be provided if you've fixed how transactions are registered?  Maybe optional would be a better version.